clients/web/static/girder-swagger.js

```diff
@@ -1,6 +1,11 @@
 $(function () {
+    var apiRoot = $('#g-global-info-apiroot').text().replace(
+        '%HOST%', 'http://' + window.location.host);
```

Comment:
This replace only occurs in tests, not in actual deployment.  We don't
currently test the swagger docs (but we should -- see issue #320), so this
doesn't matter yet.

On Fri, Apr 3, 2015 at 10:25 AM, Patrick Reynolds <notifications@github.com>
wrote:

> In clients/web/static/girder-swagger.js
> <https://github.com/girder/girder/pull/788#discussion_r27731665>:
>
> > @@ -1,6 +1,11 @@
> >  $(function () {
> > +    var apiRoot = $('#g-global-info-apiroot').text().replace(
> > +        '%HOST%', 'http://' + window.location.host);
>
> What happens if we're https only?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/girder/girder/pull/788/files#r27731665>.
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439